app/Http/Controllers/Cabinet/ConsultsController.php

```diff
@@ -47,9 +47,9 @@ public function create()
      * @return Response
      */
 
-    public function store(ConsultValid $request)
+    public function store(Request $request)
```

Comment:
Why you change back to Request?

Reply:
Because validator doesn't see the fields